packages/react-components/react-carousel/library/src/components/useEmblaCarousel.ts

```diff
@@ -305,6 +306,10 @@ export function useEmblaCarousel(
       },
       plugins,
     );
+
+    return () => {
+      onDragUnmount.current?.();
+    };
```

Comment:
I don't think that it's needed as we call `emblaApi.current?.destroy()`. In which scenario it's needed?

Reply:
This was overkill for when listeners were attached to the document instead of element, removed 👍